qa/ccs-rolling-upgrade-remote-cluster/build.gradle

```diff
@@ -19,9 +18,7 @@ dependencies {
   testImplementation project(':client:rest-high-level')
 }
 
-for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible) {
-  String bwcVersionStr = bwcVersion.toString()
-  String baseName = "v" + bwcVersionStr
+BuildParams.bwcVersions.withWireCompatiple() { bwcVersion, baseName ->
```

Comment:
Can't we omit the parens and make this `withWireCompatible {`?

Reply:
indeed. polished that here and in other locations